DataViews/Program.cs

```diff
@@ -89,7 +90,11 @@ public static async Task<bool> MainAsync(bool test = false)
                 var dataService = sdsService.GetDataService(tenantId, namespaceId);
                 var tableService = sdsService.GetTableService(tenantId, namespaceId);
 
-                var dataviewServiceFactory = new DataViewServiceFactory(new Uri(resource), new AuthenticationHandler(uriResource, clientId, clientSecret));
+                var authHandler = new AuthenticationHandler(uriResource, clientId, clientSecret);
+                var verbosityHandler = new VerbosityHeaderHandler(true); // Initialize the verbosity setting to 'verbose'
+                DelegatingHandler[] handlers = { authHandler, verbosityHandler };
```

Comment:
Since the DataViewServiceFactory constructor specifies an array of handlers as _params_ you can just pass them all in there, for example
var dataviewServiceFactory = new DataViewServiceFactory(new Uri(resource), authHandler, verbosityHandler);

Reply:
good call. I added them in line. I also didnt name the authHandler anymore since it doesn't need to be accessed later by name, but kept the verbosityHandler as is